modules/openshift-cluster-maximums-environment.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/planning-your-environment-according-to-object-maximums.adoc
+
+[id="cluster-maximums-environment_{context}"]
+= {product-title} environment and configuration on which the cluster maximums are tested
+
+AWS cloud platform:
+
+[options="header",cols="7*"]
+|===
+| Node |Flavor |vCPU |RAM(GiB) |Disk type|Disk size(GiB)/iops | Count
+
+| Master/Etcd footnoteref:[master/etcdnode, io1 disk with 3000 iops is used for master/etcd nodes as etcd is i/o intensive and latency sensitive.]
+| r5.4xlarge
+| 16
+| 128
+| io1 
+| 220G / 3000
+| 3
+
+| Infra footnoteref:[infranodes,Infra nodes are used to host openshift-monitoring, ingress and router components to make sure they have enough resources to run at large scale. Procedure on creating Infra machinesets can be found at https://docs.openshift.com/container-platform/4.3/machine_management/creating-infrastructure-machinesets.html.]
+| m5.12xlarge
+| 48
+| 192
+| gp2 
+| 100G 
+| 3
+
+| Workload footnoteref:[workload, Workload node is dedicated to run performance/scalability workload generators.]
+| m5.4xlarge
+| 16
+| 64
+| gp2 
+| 500G footnoteref:[disk, Larger disk size is used to have enough space to store large amounts of data collected during the performamce/scalability test run.]
```

Comment:
Now it looks like this footnote ID "disk" is tripping the build up. It's likely duplicated somewhere, so maybe try renaming it to disk-size or something of the sort